Lib/test/test_embed.py

```diff
@@ -1641,6 +1641,27 @@ def test_frozenmain(self):
         """).lstrip()
         self.assertEqual(out, expected)
 
+    @unittest.skipUnless(hasattr(sys, 'gettotalrefcount'),
+                         '-X showrefcount requires a Python debug build')
+    def test_no_memleak(self):
+        # bpo-1635741: Python must release all memory at exit
+        cmd = [sys.executable, "-I", "-X", "showrefcount", "-c", "pass"]
+        proc = subprocess.run(cmd,
+                              stdout=subprocess.PIPE,
+                              stderr=subprocess.STDOUT,
+                              text=True)
+        self.assertEqual(proc.returncode, 0)
+        out = proc.stdout.rstrip()
```

Comment:
Why not this?

```suggestion
        rc, _, out = assert_python_ok("-I", "-X", "showrefcount", "-c", "pass")
        self.assertEqual(rc, 0)
        out = out.decode()
```

(You'd need `from test.support.script_helper import assert_python_ok`)

Reply:
assert_python_ok() adds arguments to the command line. Here I really want to control the exact command line. Sadly, Python still leaks some references depending on the exact command line.